src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -164,7 +164,7 @@ def _get_default_install_location():
             return None
         install_location = os.path.join(home_dir, r'.azcopy\azcopy.exe')
     elif system in ('Linux', 'Darwin'):
-        install_location = '/usr/local/bin/azcopy'
+        install_location = '~/bin/azcopy'
```

Comment:
Please expand the "~" through `expanduser`, as the underneath file system doesn't recognize "~"